cli/cmd/sync.go

```diff
@@ -28,12 +28,13 @@ cloudquery sync ./directory ./aws.yml ./pg.yml
 func NewCmdSync() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "sync [files or directories]",
-		Short:   fetchShort,
-		Long:    fetchShort,
-		Example: fetchExample,
+		Short:   syncShort,
+		Long:    syncShort,
+		Example: syncExample,
 		Args:    cobra.MinimumNArgs(1),
-		RunE:    sync,
+		RunE:    logErrors(sync, "Sync failed with errors"),
```

Comment:
can we have this just in `main` this way we don't need to wrap every cobra command. also Im not sure it's super important to log this as if just exits then usually the container will exit with -1 and it will print the error but anyway I think it's fine to also print it to the logs just let's do it in one location.

Reply:
I'm not that familiar with cobra, but I'm sure how we can do it from main, because the error from `runE` doesn't seem to get returned to us anywhere? Cobra just prints it out to stdout. But I can look into it some more

Reply:
Ah okay nvm, I see now. Looks like it is being logged from `main.go` already, just without the context of what command was run. I'll remove it from this PR